wallet/wallet-cli-commands/src/lib.rs

```diff
@@ -62,6 +62,36 @@ pub enum WalletManagementCommand {
         /// Passphrase along the mnemonic
         #[arg(long = "passphrase")]
         passphrase: Option<String>,
+
+        /// Create a wallet using a connected hardware wallet. Only the public keys will be kept in
+        /// the software wallet
+        #[arg(long, conflicts_with_all(["mnemonic", "passphrase"]))]
```

Comment:
whether_to_store_seed_phrase was a required parameter I am not sure if we should break backwards compatibility or not, that is why I left it alone, and just check that it is set to false.

Reply:
Should then`whether_to_store_seed_phrase` be marked with `required_unless_present("hardware_wallet")` or something like that? So people won't have to type it every time for hardware.

Reply:
> it also conflicts with whether_to_store_seed_phrase

I guess this comment is still relevant.

Also, I suggest updating descriptions for all options that don't work with "--hardware_wallet" so that they explicitly mention this.